### PR TITLE
Dedup Keras LearningRateWarmupCallback log after gradual learning rate warmup

### PR DESCRIPTION
**Pull request by ChaiBapchya**
## Checklist before submitting

- [x] Did you read the [contributor guide](https://github.com/horovod/horovod/blob/master/CONTRIBUTING.md)?
- [x] Did you update the docs?
- [ ] Did you write any tests to validate this change?  
- [ ] Did you update the [CHANGELOG](https://github.com/horovod/horovod/blob/master/CHANGELOG.md), if this change affects users?

## Description
print learning rate only for 0th rank since lr is same across all ranks

Fixes # (issue).

## Review process to land 

1. All tests and other checks must succeed.
2. At least one member of the [technical steering committee](https://github.com/horovod/horovod/blob/master/CONTRIBUTING.md) must review and approve.
3. If any member of the technical steering committee requests changes, they must be addressed.

### CODE REVIEW

**Comment by ChaiBapchya** [comment]
cc @tgaddair

**Review by tgaddair — APPROVED** [review]
LGTM, thanks for the PR!
